profiles/bicycle.lua

```diff
@@ -544,6 +544,7 @@ function turn_function(turn)
      turn.duration = turn.duration + profile.traffic_light_penalty
   end
   if properties.weight_name == 'cyclability' then
+      turn.weight = turn.duration
```

Comment:
i find this a bit hard to understand. if you don't set the weight, i guess the duration is used? so what happens when we set weight=duration?